harvard-university-for-the-creative-arts.csl

```diff
@@ -223,19 +260,30 @@
         <text variable="number" suffix="."/>
       </else-if>
       <else-if type="motion_picture broadcast">
-        <text variable="medium" prefix="[" suffix="] "/>
+        <choose>
+          <if match="any" variable="collection-title container-title">
+            <text term="in" text-case="capitalize-first" suffix=": "/>
+            <text variable="collection-title" font-style="italic" suffix="."/>
+            <text variable="container-title" font-style="italic" suffix="."/>
```

Comment:
I have removed motion picture from this section. I have left broadcast with the variable though as we have instances of students referencing sections within broadcasts, e.g. a tv programme with different reports or sections on different topics.
If this isn't ok let me know